package.json

```diff
@@ -28,12 +28,11 @@
     "licensee"
   ],
   "devDependencies": {
-    "aud": "^2.0.4",
-    "rimraf": "^3.0.2",
+    "rimraf": "^6.0.1",
     "run-parallel": "^1.2.0",
     "spawn-sync": "^2.0.0",
-    "standard": "^14.3.4",
-    "tap": "^16.3.0"
+    "standard": "^17.1.0",
+    "tap": "^21.0.1"
```

Comment:
this too

Reply:
this is a devDep so not a breaking change .. ?

Reply:
Newer Standard will complain and `exit 1` over our `vars` and other anachronisms.

Never `tap` versions use a lot of new Node.js test features.

Reply:
Tests need to run on every supported node (not standard, but tap)